Tests/Tests.csproj

```diff
@@ -4,6 +4,8 @@
     <TargetFramework>net5.0</TargetFramework>
     <RootNamespace>RhinoMocksToMoqRewriter.Tests</RootNamespace>
     <IsPackable>false</IsPackable>
+    <LangVersion>8</LangVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
   <ItemGroup>
```

Comment:
Upgrade `NUnit` to `3.13.1`
Upgrade `NUnit3TestAdapter` to `3.17.0`
Upgrade `Microsoft.NET.Test.Sdk` to `16.8.3`